package.json

```diff
@@ -20,7 +20,7 @@
   },
   "homepage": "https://github.com/Irina-pi/if-js#readme",
   "devDependencies": {
-    "eslint": "^8.23.0",
+    "eslint": "8.22.0",
```

Comment:
What wrong with the previous?

Reply:
when I created a new branch, WebShtorm offered to automatically install ESlintrc and apparently this version.